src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Square extends Figure implements FindArea {
+    private int side;
+
+    public Square() {
+        super();
+        this.side = new Random().nextInt(1, 10);
+    }
+
+    @Override
+    public double getArea() {
+        return Math.pow(side, 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nSquere "
```

Comment:
There is a typo in the string 'Squere'. It should be corrected to 'Square' to accurately represent the figure being drawn.